theano/gradient.py

```diff
@@ -556,8 +558,12 @@ def handle_disconnected(var):
         rval = tuple(rval)
     elif not using_list:
         rval, = rval
+    t1 = time.time()
+    global grad_time
+    grad_time = t1 - t0
```

Comment:
fixed

On Mon, Feb 9, 2015 at 6:37 PM, abergeron <notifications@github.com> wrote:

> In theano/gradient.py
> <https://github.com/Theano/Theano/pull/2485#discussion_r24376866>:
>
> > @@ -556,8 +558,12 @@ def handle_disconnected(var):
> >          rval = tuple(rval)
> >      elif not using_list:
> >          rval, = rval
> > +    t1 = time.time()
> > +    global grad_time
> > +    grad_time = t1 - t0
>
> This will not compute the total time, but rather the time of the last call.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2485/files#r24376866>.
>